.gitattributes

```diff
@@ -1,6 +1,8 @@
 # Auto detect text files and perform LF normalization
 * text=auto
 
+*.fs  text eol=crlf
```

Comment:
Are we really sure we want this for all `*.fs` files?
And if so, would `lf` not make more sense?
If you are keeping this, I would include `end_of_line = crlf` as well in https://github.com/dotnet/fsharp/blob/main/.editorconfig.

Reply:
that will be removed, I was testing hashing